src/dedupe-hub/api/Dockerfile

```diff
@@ -7,3 +7,4 @@ COPY ./requirements.txt ./requirements.txt
 RUN pip install --no-cache-dir --upgrade -r requirements.txt
  
 COPY ./app ./app
+COPY ./tests ./tests
```

Comment:
It isn't good to have tests in production... Still they go to the image everywhere including Django. So I think it worth creating ticket regarding this.

Reply:
Yes, we have same situation on Django, need to think how to hide it for production & create a ticket for it.